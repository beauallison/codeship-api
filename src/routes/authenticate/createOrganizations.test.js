const createOrganizations = require('./createOrganizations');
const status200 = require('../__fixtures__/authenticate-status200.json');

describe('routes/authenticate/createOrganizations()', () => {
  it('should create organizations', () =>
    expect(createOrganizations(status200.response.organizations)).toMatchSnapshot());

  it('should return an empty Map on missing data', () =>
    expect(createOrganizations()).toMatchSnapshot());
});
