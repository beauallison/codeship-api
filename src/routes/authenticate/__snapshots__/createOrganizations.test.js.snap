// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`routes/authenticate/createOrganizations() should create organizations 1`] = `
Object {
  "Codeship": Object {
    "scopes": Array [
      "build.read",
      "project.read",
    ],
    "uuid": "dc37c39f-731e-4100-b8bf-69dae9eb14aa",
  },
  "Codeship Test Org": Object {
    "scopes": Array [
      "build.read",
      "build.write",
      "project.read",
      "project.write",
    ],
    "uuid": "dc37c39f-731e-4100-b8bf-69dae9eb14ab",
  },
}
`;

exports[`routes/authenticate/createOrganizations() should return an empty Map on missing data 1`] = `Object {}`;
